classes/profile_helper.php

```diff
@@ -104,51 +123,38 @@ public static function get_min_duration_for_group_and_reason(string $group, int
      * @return float duration (as seconds) of the fastest profile for a given reason.
      */
     public static function get_min_duration_for_reason(int $reason, bool $usecache = true): float {
-        global $DB;
-
-        $reasonstr = profile::REASON_STR_MAP[$reason];
-        $quota = (int) get_config('tool_excimer', "num_$reasonstr");
+        $quota = self::$quotas[$reason];
 
-        $cachekey = 'profile_type_' . $reason . '_min_duration_s';
-        $result = get_config('tool_excimer', $cachekey);
+        $cachekey = '__all__';
```

Comment:
Would this be better if we change this to a class constant, and update all relevant references to match?